### ISSUE TITLE
Upgrade jss to >9.7.0

### ISSUE THREAD

**Issue by codler** [cssinjs/react-jss#210]
Upgrade jss to latest jss to prevent this warning in Yarn
```
warning "react-jss > jss-preset-default@4.3.0" has incorrect peer dependency "jss@^9.7.0".
warning "react-jss > jss-preset-default > jss-camel-case@6.1.0" has incorrect peer dependency "jss@^9.7.0".
warning "react-jss > jss-preset-default > jss-default-unit@8.0.2" has incorrect peer dependency "jss@^9.4.0".
warning "react-jss > jss-preset-default > jss-expand@5.1.0" has incorrect peer dependency "jss@^9.4.0".
warning "react-jss > jss-preset-default > jss-extend@6.2.0" has incorrect peer dependency "jss@^9.7.0".
```

**Comment by kof** [comment]
released 8.3.4, is it gone now?

**Comment by codler** [comment]
yes, thanks!
